utils/docker/Dockerfile.centos:7

```diff
@@ -49,26 +49,44 @@ ARG NOBUILD
 # use same UID as host and default value of 1000 if not specified
 ARG UID=1000
 ARG DONT_USE_RPMS=true
+ARG OMPI=unknown
 
 
 # for good measure, clean the metadata
 RUN yum clean metadata
 
 # Install basic tools
 RUN yum -y install epel-release
-RUN yum -y install git gcc gcc-c++ make cmake golang libtool scons boost-devel   \
-                   libuuid-devel openssl-devel libevent-devel libtool-ltdl-devel \
-                   libcmocka libcmocka-devel readline-devel                      \
-                   doxygen pandoc flex patch nasm yasm ninja-build meson         \
-                   CUnit-devel libaio-devel python-pep8 lcov                     \
-                   python clang-analyzer sg3_utils libiscsi-devel                \
-                   numactl-devel doxygen graphviz pylint python2-pygithub        \
-                   python34 python34-pylint python34-PyYAML.x86_64 PyYAML        \
-                   python2-paramiko python34-paramiko valgrind rsync             \
-                   libyaml-devel file ShellCheck yum-plugin-copr
+RUN yum -y install \
+           git gcc gcc-c++ make cmake golang libtool boost-devel   \
+           libuuid-devel openssl-devel libevent-devel              \
+           libtool-ltdl-devel libcmocka-devel readline-devel       \
+           doxygen pandoc flex patch nasm yasm ninja-build meson   \
+           CUnit-devel libaio-devel clang-analyzer sg3_utils       \
+           libiscsi-devel numactl-devel graphviz                   \
+           libyaml-devel                                           \
+           scons pylint file ShellCheck python-pep8 lcov  python   \
+           python2-pygithub yum-plugin-copr
+
+# external deps build/repos
+# sadly this doesn't work
+#RUN yum-config-manager --save --setopt=*_job_daos-stack_job_${ext}_job_master_lastSuccessfulBuild_artifact_artifacts_.gpgcheck=0
+RUN if $DONT_USE_RPMS; then exit 0; fi;                                                                                                                     \
+for ext in openpa libfabric mercury pmix ompi; do                                                                                                                     \
+    yum-config-manager --add-repo=https://build.hpdd.intel.com/job/daos-stack/job/${ext}/job/master/lastSuccessfulBuild/artifact/artifacts/;            \
```

Comment:
I had no idea we were building RPMs for these packages, when/where was this work done?

Reply:
I've done it over the period of a couple of days as a background task while I wait for builds and tests to complete.

FWIW, CaRT builds in 4 minutes with RPMs installed rather than the 30 or so it takes without them.

Reply:
> FWIW, CaRT builds in 4 minutes with RPMs installed rather than the 30 or so it takes without them.

That's great progress!